### DISCUSSION TITLE
Example of a Rank Vote

### DISCUSSION THREAD

**Discussion by luyzdeleon** [pokt-network/organization#80] [Rank Votes] [answered]
Please rank vote all of the Proposals in this category: https://github.com/pokt-network/organization/discussions/categories/proposals. @pokt-network/engineering-member

**Comment by oten91** [comment]
1.  https://github.com/pokt-network/organization/discussions/79
2. https://github.com/pokt-network/organization/discussions/82

**Comment by luyzdeleon** [comment, marked as answer]
The result for the Rank Vote is:

1. XXX
2. YYY
3. ZZZ
